log_injection/VE7LTX_Personal.ai-Discord/d3.py

```
@@ -0,0 +1,130 @@
+    try:
+        response = requests.post(base_url, headers=headers, json=payload, timeout=60)
+        response.raise_for_status()
+        response_data = response.json()
+        
+        logging.info(f"Received response from AI: {response_data}")
+
+        ai_message = response_data.get('ai_message', '')
+        ai_score = float(response_data.get('ai_score', 0))
+        new_session_id = response_data.get('SessionId', '')
+        
+        return ai_message, ai_score, new_session_id
```

Ruff (0.13.3)

Consider moving this statement to an `else` block

(TRY300)